modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -291,50 +299,56 @@ function SubjectMember (props) {
         }
       </Grid>
       <Grid item>
-      {tableData ?
-          (<Grid container spacing={3}>
-            {tableData.map( (entry, i) => {
-              return(
-                <Grid item lg={12} xl={6} key={`${entry.questionnaire["jcr:uuid"]}-${i}`}>
-                  <Card className={classes.subjectCard}>
-                    <CardHeader
-                      title={
-                        <React.Fragment>
-                          <Button size={buttonSize} className={classes.subjectFormHeaderButton}>
-                            {entry.questionnaire["@name"]}
-                          </Button>
-                          {entry["statusFlags"].map((status) => {
-                            return <Chip
-                              key={status}
-                              label={wordToTitleCase(status)}
-                              className={`${classes.subjectChip} ${classes[status + "Chip"] || classes.DefaultChip}`}
-                              size="small"
-                            />
-                          })}
-                        </React.Fragment>
-                      }
-                      className={classes.subjectFormHeader}
-                      action={
-                        <DeleteButton
-                          entryPath={entry["@path"]}
-                          entryName={`${data && data.identifier ? data.identifier : id}: ${entry.questionnaire["@name"]}`}
-                          entryType="Form"
-                          warning={entry ? entry["@referenced"] : false}
-                        />
-                      }
-                    />
-                    <CardContent>
-                      <FormData formID={entry["@name"]} maxDisplayed={maxDisplayed}/>
-                      <Link to={"/content.html" + entry["@path"]}>
-                        <Typography variant="body2" component="p">See More...</Typography>
-                      </Link>
-                    </CardContent>
-                  </Card>
-                </Grid>
-              )
-            })}
-          </Grid>
-          ) : ""
+        { subjectGroups &&
+          Object.keys(subjectGroups).map( (questionnaireName, j) => {
+            return(<>
+	          <Typography variant="h4" className={classes.subjectQuestionnaireSubheader}>
+		        {questionnaireName}
+		      </Typography>
+		      <Grid container spacing={3}>
+		        { subjectGroups[questionnaireName].map( (entry, i) => {
+	              return(
+	                <Grid item lg={12} xl={6} key={`${entry.questionnaire["jcr:uuid"]}-${i}`}>
+	                  <Card className={classes.subjectCard}>
+	                    <CardHeader
+	                      title={
+	                        <React.Fragment>
+	                          <Button size={buttonSize} className={classes.subjectFormHeaderButton}>
+	                            {moment(entry["jcr:created"]).format("yyyy-MM-dd")}
```

Comment:
I think `dd` is for day of the week, I see `2020-11-WE`. Maybe `DD` is for day of the month in `moment.js`?
```suggestion
	                            {moment(entry["jcr:created"]).format("yyyy-MM-DD")}
```